contracts/HypercertMinterV0.sol

```diff
@@ -159,74 +159,57 @@ contract HypercertMinterV0 is
         require(data.length > 0, "Mint: input data empty");
 
         // Parse data to get Claim
-        uint256 _v = version();
+        (Claim memory claim, string memory uri_) = _parseData(data);
 
-        (
-            bytes32[] memory _rights,
-            bytes32[] memory _workScopes,
-            bytes32[] memory _impactScopes,
-            uint256[2] memory _workTimeframe,
-            uint256[2] memory _impactTimeframe,
-            address[] memory _contributors,
-            string memory _uri
-        ) = abi.decode(data, (bytes32[], bytes32[], bytes32[], uint256[2], uint256[2], address[], string));
-
-        bytes32 _claimHash = keccak256(abi.encode(_workTimeframe, _workScopes, _impactTimeframe, _impactScopes, _v));
+        require(claim.workTimeframe[0] < claim.workTimeframe[1], "Mint: invalid workTimeframe");
```

Comment:
Could this also be equal? Given that we'll be looking at a date level, we could expect a timestamp for beginning and end of day respectively. Another option is that they're identical

Reply:
Yup, good point. Updated.